--- include/cpr/cpr.h
+++ include/cpr/cpr.h
@@ -11,1 +11,0 @@
-#include "cpr/cprver.h"

--- include/cpr/filesystem.h
+++ include/cpr/filesystem.h
@@ -4,0 +4,10 @@
+////////////////////////////////////////
+// <Begin Hedron Patch>
+
+#include "backport/filesystem.hpp"
+#endif
+#if false  // Ignore rest of file, without making the patch brittle by containing the rest of the file.
+
+// <End Hedron Patch>
+////////////////////////////////////////
+
